db/migrate/20220525122759_init_schema.rb

```diff
@@ -0,0 +1,235 @@
+class InitSchema < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension 'pgcrypto'
+
+    create_table "add_ons", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+      t.uuid "organization_id", null: false
+      t.string "name", null: false
+      t.string "code", null: false
+      t.string "description"
+      t.bigint "amount_cents", null: false
+      t.string "amount_currency", null: false
+      t.datetime "created_at", null: false
+      t.datetime "updated_at", null: false
+      t.index ["organization_id", "code"], name: "index_add_ons_on_organization_id_and_code", unique: true
+      t.index ["organization_id"], name: "index_add_ons_on_organization_id"
+    end
+
+    create_table "applied_coupons", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)